vizro-core/examples/_chart-gallery/pages/examples/column_time.py

```diff
@@ -0,0 +1,36 @@
+import pandas as pd
+import vizro.models as vm
+import vizro.plotly.express as px
+from vizro import Vizro
+from vizro.models.types import capture
+
+gapminder = px.data.gapminder()
+
+
+@capture("graph")
+def column(data_frame: pd.DataFrame, x: str, y: str):
```

Comment:
```suggestion
def time_column(data_frame: pd.DataFrame, x: str, y: str):
```
Or `categorical_column` or whatever you like. `column` just feels too generic here.